packages/cli/src/cmds/prune/prune.ts

```diff
@@ -46,31 +46,60 @@ function parseSize(size: string) {
 }
 
 export default {
-  command: 'prune <file> <size>',
+  command: 'prune <file>',
 
-  describe: 'Prune an appmap file down to the given size',
+  describe: 'Make an appmap file smaller by removing events',
 
   builder: (argv: Yargs.Argv) => {
     argv.option('output-dir', {
-      describe: 'Specifies the output directory. Pruned files will be written here.',
+      describe: 'Specifies the output directory',
       type: 'string',
       default: '.',
       alias: 'o',
     });
+
+    argv.option('json', {
+      describe: 'Return the exclusions as json',
+      type: 'boolean',
+    });
+
     argv.positional('file', {
       describe: 'AppMap to prune',
     });
-    argv.positional('size', {
+
+    argv.option('size', {
       describe: 'Prune input file to this size',
+      default: '15mb',
+      type: 'string',
+      alias: 's',
+    });
+
+    argv.option('events', {
+      describe: 'Events to remove from the map',
```

Comment:
How would I use this option?

Reply:
Let's expand this to use a `base64url` encoded [`AppMapFilter`](https://github.com/getappmap/appmap-js/blob/main/packages/models/src/appMapFilter.js#L75) - the same as the filters we apply within an AppMap. The logic for applying these filters is already contained within, but it does expect a normalized AppMap as input.